### PR TITLE
New jbrowse and python-tripal versions

### PR DESCRIPTION
**Pull request by abretaud**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by abretaud** [comment]
Anybody have any clue why travis is red? I don't understand the error...

File: recipes/jbrowse/1.12.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package:
+  name: jbrowse
+  version: "1.12.1"
+build:
+  number: 4
```
Comment (bgruening) [inline recipes/jbrowse/1.12.1/meta.yaml]: should this start with 0

File: recipes/jbrowse/meta.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 package:
   name: jbrowse
-  version: "1.12.1"
+  version: "1.12.3"
 build:
-  number: 3
+  number: 1
```
Comment (bgruening) [inline recipes/jbrowse/meta.yaml]: 0 ?

**Comment by bgruening** [comment]
This was an failure on our site in the build system. But now we have a new one.
Do you know where `/anaconda/conda-bld/osx-64/python-tripal-1.5-py27_0.tar.bz2` comes from?

**Comment by abretaud** [comment]
Ok, good!
For `/anaconda/conda-bld/osx-64/python-tripal-1.5-py27_0.tar.bz2` where do you see it? I don't know, there is a recipe for version 1.5 and building is enabled for osx and py27, so maybe it's normal?

**Comment by abretaud** [comment]
O, still red 😭

**Pull request merged**